### ISSUE TITLE
Hyperparameter tuning

### ISSUE THREAD

**Issue by goutamyg** [researchmm/LightTrack#32]
Hi! Thank you for publishing your code.

In your paper, you have mentioned various training-related choices/hyperparameters (e.g., learning rate, number of epochs). Your tracker evaluation script has parameters related to the post-processing of bounding-box output by the tracker (penalty_k, window_influence, lr).

Can you please suggest how did you arrive at these hyperparameter values? Was it fine-tuned using the test-set itself?
